### ISSUE TITLE
Date and hour config, bug on registration event.

### ISSUE THREAD

**Issue by leonardopaiva** [phonegap/phonegap-plugin-push#114]
Hello, i dont know if this is an issue, but maybe could be better documented... The registration event wasnt working on my "tablet xoom2 android 4.0.3", i tried alot, and found that the problem was with the date and hour configuration from my tablet. 

Anyway, congratulations for the work.

**Comment by macdonst** [comment]
@leonardopaiva I need more information. What was the problem with date/time? How did you resolve it?

**Comment by leonardopaiva** [comment]
Hello @macdonst,  as requested posting more details...

The problem happens when my tablet have a wrong date configured and have no connection to internet. Registration or error, none of the events are called.

I changed the date of my tablet to reproduce the problem...

changed the year to 2010 - bug - registration or error, none of the events was called.
changed the year to 2014 - bug - registration or error, none of the events was called.
changed the year to 2015 - ok - registration event called (need internet to work).
2016  or more, i cant test more than 2015, because my tablet changes the date to 2015 automaticaly.

I`m using this example https://github.com/phonegap/phonegap-plugin-push/blob/master/example/www/js/index.js

just calling alert on registration and error events, using the same sender id 741175631277.

**Comment by macdonst** [comment]
@leonardopaiva Yeah, it seems like it should report back as an error but it depends on how Google responds to the registration event. I'll try and take a look at it soon.

**Comment by leonardopaiva** [comment]
ok @macdonst , i`m here if you need help to test.
